e2e/config/config.go

```diff
@@ -31,6 +35,15 @@ type Repo struct {
 	Branch string
 }
 
+// DRNamespaces defines the user-configurable namespace names required
+// for RamenDR to run e2e tests.
+type DRNamespaces struct {
```

Comment:
DRNamespaces is good name, but everying here is in the context of DR, so we can remove the DR prefix.